src/elements/checkbox-element.js

```diff
@@ -0,0 +1,7 @@
+import BooleanElement from './boolean-element';
+
+export default class CheckBoxElement extends BooleanElement {
```

Comment:
`CheckBox` -> `Checkbox` everywhere since it is one word?

Reply:
I am fine with that as long as we are consistent.